ql/src/test/results/clientpositive/llap/show_functions.q.out

```diff
@@ -218,6 +218,12 @@ hash
 hex
 histogram_numeric
 hour
+iceberg_bucket
```

Comment:
hm, why output changed?

Reply:
because of the test-scoped iceberg handler dependency
it was needed to be able to run query_history.q in the itests/qtest module (and not in the separate iceberg related modules

https://github.com/apache/hive/pull/5613/files#diff-c0bf514f60161dcf9a3b90898274ad5a3e550f0a1d0169ca8b0521a84382cb27R289

```
    <dependency>
      <groupId>org.apache.hive</groupId>
      <artifactId>hive-iceberg-handler</artifactId>
      <version>${project.version}</version>
      <scope>test</scope>
    </dependency>
```